tests/RestfulListTestCase.test

```diff
@@ -427,6 +436,19 @@ class RestfulListTestCase extends RestfulCurlBaseTestCase {
 
     $result = $handler->get('', $request);
     $this->assertEqual(count($result), 2, 'List filtered by the "author" entity metadata wrapper property, which maps to the "uid" DB column name.');
+
+    $request = array('filter' => array(
+      'static' => array(
+        'value' => 0,
+      )
+    ));
+    try {
+      $handler->get('', $request);
+      $this->fail('Exception not thrown for invalid filter.');
+    }
+    catch (\RestfulBadRequestException $e) {
```

Comment:
If there is an uncatched exception, the tests will fail and simpletest will provide more information than with a cached exception.